### PR TITLE
feat(editor): Removing `ph-no-capture` class from some elements

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/design-system/src/components/N8nInput/Input.vue [hunk 1]
```diff
@@ -85,11 +85,14 @@ export default defineComponent({
...
 		classes(): string[] {
+			const classes = [];
 			if (this.size === 'xlarge') {
-				return ['xlarge'];
+				classes.push('xlarge');
 			}
-
-			return [];
+			if (this.type === 'password') {
+				classes.push('ph-no-capture');
```
Comment (mutdmour) [inline packages/design-system/src/components/N8nInput/Input.vue:93]: we need tests for this somewhere?

File: packages/editor-ui/src/views/SettingsApiView.vue [hunk 2]
```diff
@@ -36,7 +36,7 @@
 						{{ $locale.baseText('generic.delete') }}
 					</n8n-link>
 				</span>
-				<div class="ph-no-capture">
+				<div>
```
Comment (mutdmour) [inline packages/editor-ui/src/views/SettingsApiView.vue:39]: seems like this one we should keep redacted...

Comment (mutdmour) [reply packages/editor-ui/src/views/SettingsApiView.vue:39]: and test for it

**Review by mutdmour — COMMENTED** [review]
nice

File: packages/design-system/src/components/N8nInput/Input.vue (hunk 1 above)
Comment (MiloradFilipovic) [reply packages/design-system/src/components/N8nInput/Input.vue:93]: Yup. Will add those.

File: packages/design-system/src/components/N8nInput/Input.vue [hunk 3]
```diff
@@ -85,11 +85,14 @@ export default defineComponent({
...
 		},
 		classes(): string[] {
+			const classes = [];
 			if (this.size === 'xlarge') {
-				return ['xlarge'];
+				classes.push('xlarge');
 			}
-
-			return [];
+			if (this.type === 'password') {
```
Comment (mutdmour) [inline packages/design-system/src/components/N8nInput/Input.vue:92]: not every input in credentials modal is a password input.. should we handle those seperately.. for example sometimes api keys are plain text inputs..

Comment (MiloradFilipovic) [reply packages/design-system/src/components/N8nInput/Input.vue:92]: True, but we need to hide password inputs no matter where they are and it's done here. All other inputs in credentials modals are hidden by [this](https://github.com/n8n-io/n8n/blob/2e94e4ff652a29930bf8835ad31954e2cffaf00d/packages/editor-ui/src/components/ParameterInput.vue#L833).
Example:
image

Comment (mutdmour) [reply packages/design-system/src/components/N8nInput/Input.vue:92]: perfect

**Review by mutdmour — DISMISSED** [review]

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.1.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.1.0)

**Pull request merged**